tools/automator/automator.sh

```diff
@@ -257,7 +257,15 @@ merge() {
   git -c "user.name=$user" -c "user.email=$email" merge --no-ff -m "$title" --log upstream/"$merge_branch"
   local code=$?
   if [ "$code" -ne 0 ]; then
-    print_error "$(git status)" 1
+    export GITHUB_TOKEN="$token"
+    local issue_exists
+    issue_exists=$(gh issue list -S "Automatic merge of $merge_branch into $branch failed." -R "istio/isio" | wc -l)
+    if [ "$issue_exists" -eq 0 ]; then
+      gh issue create -b "Automatic merge of $merge_branch into $branch failed. @istio/wg-networking-maintainers" -t "Automatic merge of upstream envoy release branch failed" -l "area/networking/envoy" -R "istio/istio"
+      print_error "Conflicts detected, manual merge is required. An issue in istio/istio has been created." 0
+    else
+      print_error "Conflicts detected, manual merge is required. An issue in istio/istio already exists." 0
```

Comment:
is there any point to this else? I.e. is it cleaner to do: 

```python
  if [ "$issue_exists" -eq 0 ]; then
      gh issue create -b "Automatic merge of $merge_branch into $branch failed. @istio/wg-networking-maintainers" -t "Automatic merge of upstream envoy release branch failed" -l "area/networking/envoy" -R "istio/istio"
fi
     print_error "Conflicts detected, manual merge is required. An issue in istio/istio has been created." 0
```

Seems to be the same log message in both if and else cases

Reply:
The other message is slightly different -- it mentions that an issue already exists. I don't feel strongly about it though, can remove it if you don't think it's useful.

Reply:
Thanks :) Misread that earlier.